dist/build.gradle

```diff
@@ -79,15 +79,21 @@ task distDir(group: 'Build',
              description: "Builds a distribution directory (${project.ext.relativeDistDir})",
              dependsOn: [tasks.copyLicenses, tasks.copyLib, tasks.copyClientBinaries])
```

Comment:
You could make `distDir` depend on `distDirWithoutClientBinaries` to reduce duplication of dependencies.